packages/ozone/src/api/moderation/getRecord.ts

```diff
@@ -11,18 +11,20 @@ export default function (server: Server, ctx: AppContext) {
       const db = ctx.db
       const labelers = ctx.reqLabelers(req)
 
-      const [record, accountInfo] = await Promise.all([
-        ctx.modService(db).views.recordDetail(params, labelers),
+      const [records, accountInfos] = await Promise.all([
+        ctx.modService(db).views.recordDetail([params], labelers),
```

Comment:
Can we pluralize everything that is taking a list & returning a map?

ie `modService(db).views.recordDetails`

and `getPdsAccountInfos`